.gitignore

```diff
@@ -1,6 +1,7 @@
 *.pyc
 __pycache__/
 marimbabot_vision/scripts/data/
+marimbabot_vision/scripts/data_augmented/
```

Comment:
The upstream gut ignore features a `data_*` rule iirc. So this would be redundant